tests/ices.zunit

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env zunit
+
+@setup {
+  ZPLUGINS=$ZINIT[PLUGINS_DIR]
+
+  function _zunit_assert_not_exists() {
+    local pathname=$1 filepath
+
+    # If filepath is relative, prepend the test directory
+    if [[ "${pathname:0:1}" != "/" ]]; then
+      filepath="$testdir/${pathname}"
+    else
+      filepath="$pathname"
+    fi
+
+    [[ ! -e "$filepath" ]] && return 0
+
+    echo "'$pathname' does exist"
+    exit 1
+  }
+
+}
+
+@test 'mv' {
+  run zinit as"null" id-as"test/mv" mv"readme.md -> mv.md" for zdharma-continuum/null
+  assert $state equals 0
+  assert "$ZPLUGINS/test---mv/mv.md" is_file
+  assert "$ZPLUGINS/test---mv/mv.md" is_file
+  assert "$ZPLUGINS/test---mv/readme.md" not_exists
+}
+
+@test 'cp' {
+  run zinit as"null" id-as"test/cp" cp"readme.md -> cp.md" for zdharma-continuum/null
+  assert $state equals 0
+  assert "$ZPLUGINS/test---cp/cp.md" is_file
+  assert "$ZPLUGINS/test---cp/cp.md" is_readable
+  assert "$ZPLUGINS/test---cp/readme.md" is_file
+}
+
+@test 'atclone' {
+  run zinit as"null" id-as"test/atclone" atclone"mv readme.md atclone.md" for zdharma-continuum/null
+  assert $state equals 0
+  assert "$ZPLUGINS/test---atclone/atclone.md" is_file
+  assert "$ZPLUGINS/test---atclone/atclone.md" is_readable
+  assert "$ZPLUGINS/test---atclone/readme.md" not_exists
+}
+
+@test 'make' {
+  run zinit as"null" id-as"test/make" atclone"printf 'all:\n\ttouch whatever\n' > Makefile" make"" for zdharma-continuum/null
```

Comment:
I'm surprised this works. I'd expect it to fail as AFAIK it tries to run `make install`.

Reply:
All three places run `        command make -C "$dir" ${(@s; ;)${make#\!\!}}` so no mention of `install` (as far as I understand this line)

Reply:
Just FYI: this is the wording of an example in the readme: from my reading, it's in line with the code, i.e. if `install` is not the default make target, you have explicitly supply the target:

```shell
# Scripts built at install (there's single default make target, "install",
# and it constructs scripts by `cat'ing a few files). The make'' ice could also be:
# `make"install PREFIX=$ZPFX"`, if "install" wouldn't be the only default target.
zinit ice as"program" pick"$ZPFX/bin/git-*" make"PREFIX=$ZPFX"
zinit light tj/git-extras
```